js/src/encoding-test.js

```diff
@@ -300,7 +301,7 @@ suite('Encoding', () => {
     assert.deepEqual(encoding, w.toArray());
 
     const r = new TestReader(encoding);
-    const dec = new ValueDecoder(r, null, staticTypeCache);
+    const dec = new ValueDecoder(r, (null: any), staticTypeCache);
```

Comment:
I'll add a comment. I didn't want to make ValueDecoder accept null but this test never uses the value reader.